README.md

```diff
@@ -80,3 +88,7 @@ module "ha_vault" {
   key_name             = "sam"
 }
 ```
+
+### Upgrade from v1.x to v2.x
```

Comment:
don't forget to also add this to the release

Reply:
sure